codex-ui/src/vue/components/alert/Alert.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div
+    ref="el"
+    :class="[
+      $style.alert,
+      $style['alert--' + props.type]
+    ]"
+    :theme-base="computedTheme"
+  >
+    <Icon
+      v-show="icon"
+      :name="icon"
+    />
+    <div>{{ props.message }}</div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { defineProps, computed, withDefaults, ref } from 'vue';
+import Icon from '../icon/Icon.vue';
+import type { AlertOptions } from './Alert.types';
+import { genId } from './constant';
+
+const el = ref<HTMLElement>();
+
+const props = withDefaults(defineProps<AlertOptions>(), {
+  id: genId(),
+  position: 'bottom-center',
+  message: '',
+  icon: '',
+  type: undefined,
```

Comment:
lets make it `default` by default

Reply:
as in defaultProps for the variable name?